README.md

```diff
@@ -19,6 +19,7 @@ Additionally, the JSON file for mainnet contains the following data to support o
 - config for the Demex Trading League competition
 - perp pool promotion parameters
 - typeform survey parameters
+- market banner parameters
```

Comment:
`parameters for information banners to be displayed on the TradingView charts on Trade UI`